week2/d03/Assignment_2_3_1/src/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+add_library(ex1 STATIC ex1.cpp)
```

Comment:
Optionally you can have these in one add_library: 
add_library(ex STATIC ex1.cpp ex2.cpp ex3.cpp etc...)